src/main/java/core/basesyntax/strategy/impl/SupplyOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationHandler implements OperationHandler {
+    private FruitStorageDao fruitStorageDao = new FruitStorageDaoImpl();
+
+    @Override
+    public void operationProcess(FruitTransaction fruitTransaction) {
+        int value = fruitStorageDao.get(fruitTransaction.getFruit())
+                + fruitTransaction.getQuantity();
+        fruitStorageDao.add(fruitTransaction.getFruit(), value);
+    }
+}

```

Comment:
Supply, return and balance services do almost the same, i think you can combine it in one service to avoid code duplication